### DISCUSSION TITLE
Stato tende invertito - HomeKit Controller

### DISCUSSION THREAD

**Discussion by Criaga** [indomus/forum#2509] [HUB: Home Assistant]
Buondì 
Tramite HomeKit controller ho integrato in HA il mio sistema Vimar VIEW wireless. Le Luci funzionano alla perfezione e anche le tende,anche se, il loro stato è invertito. 
Risultano aperte (su HA) ma sono chiuse e viceversa. 
Sull app Vimar tutto è corretto 
Come posso risolvere?

Grazie

**Comment by snakuzzo** [comment]
Controlla qui...

https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+homekit_controller%22

...se non trovi nulla, apri una issue
